src/types/response.ts

```diff
@@ -66,7 +66,8 @@ export interface ListUnspentsBackendResponse {
   amount: string,
   change: string,
   fee: string,
-  serviceFee: string
```

Comment:
a to się nie wysyła teraz w zagnieżdżonym obiekcie po zmianie z prowizjami?

Reply:
Jeszcze nie, to bedzie druga zmiana.